### PR TITLE
Adding hometown file for kentrain1

### PR DESCRIPTION
**Pull request by KenTrain1**
@RamyaYeddula what kind of food you like so I can add them to the hometown file?

### CODE REVIEW

File: kentrain1-sturgeonbay.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Sturgeon Bay
+
+## restaruant
+
```
Comment (RamyaYeddula) [inline kentrain1-sturgeonbay.md]: I really like pasta.   :+1:

**Review by RamyaYeddula — COMMENTED** [review]
Suggest some restaurants

**Comment by KenTrain1** [comment]
@RamyaYeddula  This is good, I am made changes, added a place  with pasta, let me know if this is good

**Review by githubteacher — APPROVED** [review]
This looks great :+1: :tada:

**Pull request merged**